src/writer/creator.cpp

```diff
@@ -314,8 +314,9 @@ namespace zim
 
       header->setMimeListPos( Fileheader::size );
 
-      auto titleListClusterOffset = data->mp_titleListDirent->getCluster()->getOffset();
-      header->setTitleIdxPos(titleListClusterOffset.v + data->m_titleListOffset);
+      auto cluster = data->mp_titleListDirent->getCluster();
+      header->setTitleIdxPos(
+        1 + cluster->getOffset().v + cluster->getDataOffset().v + data->m_titleListOffset);
```

Comment:
Isn't it better to hide that 1 in `Cluster::getDataOffset()`? That would be more consistent with the semantics of `Cluster::getOffset()`.

Reply:
Yes. Done.